.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Feature Request
+about: Suggest an idea for this project	about
+title: ''
+labels: kind/feature-request
+assignees: ''
+
+---
+
+**Problem**
+A clear and concise description of what the problem or the user story this feature will solve. Ex. I'm always frustrated when [...]
+
+ **User Persona**
+What user persona does this feature address? These could be multiple. If possible, define "I can" statements for the user persona in 'Problem' and/or 'Exit criteria' section.
```

Comment:
I think this should be a single persona for a specific feature. If we create an issue for a scenario that involves multiple personas, we can always update the text.